base/base.jl

```diff
@@ -4,15 +4,32 @@ type SystemError <: Exception
     prefix::AbstractString
     errnum::Int32
     extrainfo
+    @doc """
+       SystemError(prefix::AbstractString, [errno::Int32])
+
+    A system call failed with an error code (in the `errno` global variable).
+    """ ->
```

Comment:
Since this docstring doesn't apply specifically to the 3-arg constructor it would probably be better to just put it above the `type` itself. Same with the `DimensionMismatch` and `AssertionError` ones below as well.